SU2_CFD/src/solvers/CMeshSolver.cpp

```diff
@@ -189,94 +179,99 @@ CMeshSolver::CMeshSolver(CGeometry *geometry, CConfig *config) : CFEASolver(true
 
 }
 
-CMeshSolver::~CMeshSolver(void) {
```

Comment:
Moved to .hpp?

Reply:
Deleted as there is nothing to explicitly delete in this class, the compiler will create a default.